src/main/java/core/basesyntax/service/impl/FruitParserImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitParser;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitParserImpl implements FruitParser {
+    private static final String REGEX = ",";
+    private static final int INDEX_OPERATION = 0;
+    private static final int INDEX_FRUIT = 1;
+    private static final int INDEX_QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> createTransaction(String dataFromFile) {
+        return Arrays
+                .stream(dataFromFile.split(System.lineSeparator()))
+                .skip(1)
+                .map(this::fruitTransactionCreation)
+                .collect(Collectors.toList());
+    }
+
+    private FruitTransaction fruitTransactionCreation(String data) {
```

Comment:
You can split this service in two as you have two different methods, but it's not obligatory because I saw realisation similar to yours and they were approved :)